### ISSUE TITLE
Renaming and then removing this repository

### ISSUE THREAD

**Issue by greglandrum** [rinikerlab/mdfptools#8]
**TL;DR**: We're going to be renaming this respository from `mdfptools` to `mdfptools_old` today and then creating a new `mdfptools` repo which has the same contents as this but has some history edits to remove large files which were checked in at some point in the past and then removed. At some point in the next week we'll delete this repo.

**Why are we doing this?**
At some point in the early days of this repo we added some large files which required using git-lfs. After realizing that it didn't make sense to have those files in this repo we removed them. Unfortunately, the files are still in the history of the repo and cause to require git-lfs. Github has a quota for maximum free bandwidth usage for git-lfs that is used whenever anyone forks or clones this repo. With increasing usage of the repo, we're using that quota up really quickly each month and it's starting to cause problems. Since we aren't actually *using* git-lfs here, it doesn't make sense for us to pay to increase our quota.

According to the github docs, the only solution to the situation is to remove and recreate the repo. So that's what we're doing.

**Comment by greglandrum** [comment]
@AspirinCode, @tsenapathi, @zhangdachuafoodies : you have created forks of this repo, so this may affect you.
